src/xrpld/ledger/detail/ApplyStateTable.cpp

```diff
@@ -203,6 +216,13 @@ ApplyStateTable::apply(
                         prevs.emplace_back(obj);
                 }
 
+                if (isBatch && nodeType == ltACCOUNT_ROOT && batchPrev)
+                {
+                    // TODO: This could fail if the fields already exist
```

Comment:
is this TODO done? also would love to learn more about this, as the comment itself isn't too comprehensive 😃